x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/connector_types/gemini.ts

```diff
@@ -269,27 +274,28 @@ export default function geminiTest({ getService }: FtrProviderContext) {
               config: configService.get('kbnTestServer.serverArgs'),
             },
           });
-          let url: string;
+          let apiUrl: string;
           let geminiActionId: string;
 
           before(async () => {
-            url = await simulator.start();
-            geminiActionId = await createConnector(url);
+            apiUrl = await simulator.start();
+            geminiActionId = await createConnector(apiUrl);
           });
 
           after(() => {
             simulator.close();
           });
 
-          it('should invoke AI with assistant AI body argument formatted to gemini expectations', async () => {
+          // TODO to fix, we need to figure out how to mock the gcp oauth token
```

Comment:
Could you please open an issue about it and assign it to your team?

Reply:
Done! https://github.com/elastic/kibana/issues/195437

I left a comment in the code too https://github.com/elastic/kibana/pull/195308/commits/9e03463a5b5d2b6578da3bfcb2b1c57c2ffb9d0b

Thanks for the review @cnasikas !!